client/gutenberg/extensions/related-posts/constants.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+export const MAX_POSTS_TO_SHOW = 3;
+export const DEFAULT_POSTS = [
+	{
+		title: 'Big iPhone/iPad Update Now Available',
+		image: 'https://wordpress.com/calypso/images/related-posts/cat-blog.png',
+		date: 'August 3, 2018',
+		context: 'In "Mobile"',
+	},
+	{
+		title: 'The WordPress for Android App Gets a Big Facelift',
+		image: 'https://wordpress.com/calypso/images/related-posts/devices.jpg',
+		date: 'August 2, 2018',
+		context: 'In "Mobile"',
+	},
+	{
+		title: 'Upgrade Focus: VideoPress For Weddings',
+		image: 'https://wordpress.com/calypso/images/related-posts/mobile-wedding.jpg',
+		date: 'August 5, 2018',
+		context: 'In "Upgrade"',
+	},
+];
```

Comment:
These example texts and images are straight from the current version in Jetpack — since we're at it, we should probably look up some more fresh demo content. Won't stop merging though, just noting down.

Reply:
Good point 👍 

However, I'll defer that decision to @MichaelArestad and @mapk - I'm not planning any changes, as that would affect other bits that already use this content.

Also, FWIW, if we decide to change the content, we should do in all instances - both Jetpack (the plugin) and Calypso (site settings).

Reply:
That's correct @ockham.